### DISCUSSION TITLE
How to distinguish the source of the traffic sent by the bucket (minio_bucket_traffic_sent_bytes)?

### DISCUSSION THREAD

**Discussion by nowinkeyy** [minio/minio#18412] [Q&A] [answered]
I want to distinguish whether the traffic reading bucket data comes from inside the k8s cluster or outside the k8s cluster, can I do that? I look forward to your answers and thank you very much.

**Comment by harshavardhana** [comment, marked as answer]
You need to implement that yourself at a layer above. MinIO does not distinguish for where is the request coming from.

**Reply by nowinkeyy** [reply]
thank you!
